README.md

```diff
@@ -136,6 +136,7 @@ Here is a list of customizable behaviors:
 public var animateView = ZLSwipeableView.defaultAnimateViewHandler()
 public var interpretDirection = ZLSwipeableView.defaultInterpretDirectionHandler()
 public var shouldSwipeView = ZLSwipeableView.defaultShouldSwipeViewHandler()
+public var angle = CGFloat(1.0)
```

Comment:
cool! could you also add an entry at line 160 of the README with an explanation for how it works?

P.S.: this commit doesn’t link to your github profile, if that’s something you care about I’d recommend updating the git config on the machine you’re working on to include your github email address!